src/main/java/com/datadog/api/v2/openapi.yaml

```diff
@@ -3818,6 +3818,14 @@ paths:
       tags:
       - Incidents
       x-codegen-request-body-name: body
+      x-given:
```

Comment:
Should this be reset, looks like pre-commit is failing. Probably not an issue since the merge of this will include the overwritten regeneration anyway